pkg/testgridanalysis/testgridconversion/ocp_synthentic_tests.go

```diff
@@ -37,11 +37,11 @@ func (openshiftSyntheticManager) CreateSyntheticTests(rawJobResults testgridanal
 			}
 
 			syntheticTests := map[string]*synthenticTestResult{
-				testgridanalysisapi.InstallTestName:             {name: testgridanalysisapi.InstallTestName},
-				testgridanalysisapi.InstallTimeoutTestName:      {name: testgridanalysisapi.InstallTestName},
-				testgridanalysisapi.InfrastructureTestName:      {name: testgridanalysisapi.InfrastructureTestName},
-				testgridanalysisapi.FinalOperatorHealthTestName: {name: testgridanalysisapi.FinalOperatorHealthTestName},
-				testgridanalysisapi.OpenShiftTestsName:          {name: testgridanalysisapi.OpenShiftTestsName},
+				testgridanalysisapi.InstallTestName:             &synthenticTestResult{name: testgridanalysisapi.InstallTestName},
+				testgridanalysisapi.InstallTimeoutTestName:      &synthenticTestResult{name: testgridanalysisapi.InstallTestName},
+				testgridanalysisapi.InfrastructureTestName:      &synthenticTestResult{name: testgridanalysisapi.InfrastructureTestName},
+				testgridanalysisapi.FinalOperatorHealthTestName: &synthenticTestResult{name: testgridanalysisapi.FinalOperatorHealthTestName},
+				testgridanalysisapi.OpenShiftTestsName:          &synthenticTestResult{name: testgridanalysisapi.OpenShiftTestsName},
```

Comment:
looks like synthetic is still typoed here.

Reply:
Good catch, thanks!